aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -574,22 +560,49 @@ class QConvInt8 final {
       auto bias_fp32 = pack_data.bias;
       int8_t* w_data =
           reinterpret_cast<int8_t*>(weight_contig.data_ptr<c10::qint8>());
+
+      std::tie(pack_data.w_zero_points, pack_data.w_scales) =
+          make_zero_points_and_scales_tensor(weight_contig);
```

Comment:
Actually this can be done in the conv_prepack function, right?

Reply:
Yes but it is better for readability that it is here, otherwise you will have to go back and forth to understand what is happening.

Reply:
I think as long as we add a comment it should suffice. We can get `pack_data.w_scale` as before, except that it will be a tensor of scales now.

Reply:
My argument is that, we should not fill up pack_data, half in one place, half in another unless there is a good reason. It does not read well.

Reply:
I think I am gonna have to this with torchbind support landed. But I have another way of doing per channel quant that will not require requant scales to be stored in this data structure. Hopefully then it will be cleaner as well.